.env

```diff
@@ -2,6 +2,35 @@ REACT_APP_CHAIN_ID="122"
 REACT_APP_NETWORK_URL="https://rpc.fuse.io/"
 REACT_APP_ROPSTEN_NETWORK_URL=https://ropsten.infura.io/v3/2ba4786ec53f4b4aadd7ef61f37e5080
```

Comment:
We still mix approaches here by using ropsten/mainnet in one single config, but splitting bsc and bsc tesnet to prod a qa.
IMO the right thing is to have `REACT_APP_ETHEREUM_NETWORK_URL` and `REACT_APP_ETHEREUM_CHAIN_ID` and have different values for prod and qa. if the change isn't to wide I say let's do this